pages/_app.tsx

```diff
@@ -119,7 +117,8 @@ const SmallBusinessHeroApp = ({Component, pageProps}) => {
 							</div>
 							{/*  Placeholder text for logo*/}
 						</Link>
-						{/* <SvgIcon viewBox={"0 0 55 55"} fontSize={'large'}><BasketIcon /></SvgIcon> */}
+						<Link href="https://docs.google.com/forms/d/1qVNaUAvrFfpXZvRB3ZKv-1RelD6GTnbejSulJMfqUwg/viewform?edit_requested=true" target="_blank">Registriere deinen Laden</Link>
```

Comment:
Please add `rel="noopener"` to protect against attacks ([see explanation here for more details](https://developers.google.com/web/tools/lighthouse/audits/noopener))

Reply:
done